CONTRIBUTING.md

```diff
@@ -12,13 +12,19 @@ Once a commit is made to `develop`, a PR should be opened from `develop` into `m
 
 # Unit Testing
 
-TL;DR
+When contributing to WPCS, it's recommended that the project has its own copy of `phpcs`, installed via Composer:
```

Comment:
I've never actually thought about just using the dependency version of PHPCS for running tests / submitting PRs upstream to PHPCS itself.